### DISCUSSION TITLE
How about adding setup.py and setup.cfg?

### DISCUSSION THREAD

**Discussion by nim-hrkn** [Tomoki-YAMASHITA/CrySPY#22] [Ideas]
An example of setup.py
```
from setuptools import setup

if __name__ == '__main__':
    setup()

```

An example, for setup.cfg
```
[metadata]
name = CrysPy
version = 0.10.3
license = MIT License
python_requires >= 3.8

[options]
zip_safe = False
include_package_data = True
packages = CrySPY

install_requires =
  numpy
  pandas
  pyxtal >= 0.2.2
  pymatgen >= 2022
  configparser
```

**Comment by Tomoki-YAMASHITA** [comment]
Thank you for the example of setup.py. It seems easy.

I'll try to handle this in the next version update.
In addition, I plan to register with PyPI.

**Comment by nim-hrkn** [comment]
To work with other programs, it is easier to use if they are installed equivalently with other programs by setup.py.
